src/Extension/FluentSiteTreeExtension.php

```diff
@@ -14,6 +16,8 @@
  */
 class FluentSiteTreeExtension extends FluentVersionedExtension
 {
+    private static $disable_locale_published_status_message = false;
```

Comment:
I think we can flip this to truthy, given now that 4.0 lets us override truthy config with falsey. :)